conversions.go

```diff
@@ -159,15 +158,17 @@ func ECIToLookAngles(eciSat Vector3, obsCoords LatLong, obsAlt, jday float64) (l
 	top_e := -math.Sin(theta)*rx + math.Cos(theta)*ry
 	top_z := math.Cos(obsCoords.Latitude)*math.Cos(theta)*rx + math.Cos(obsCoords.Latitude)*math.Sin(theta)*ry + math.Sin(obsCoords.Latitude)*rz
 
-	lookAngles.Az = math.Atan(-top_e / top_s)
-	if top_s > 0 {
-		lookAngles.Az = lookAngles.Az + math.Pi
-	}
-	if lookAngles.Az < 0 {
-		lookAngles.Az = lookAngles.Az + 2*math.Pi
-	}
-	lookAngles.Rg = math.Sqrt(rx*rx + ry*ry + rz*rz)
-	lookAngles.El = math.Asin(top_z / lookAngles.Rg)
+	rx := ecefSat.X - ecefObs.X
```

Comment:
sorry this took a while to get to - i haven't been very active with OSS lately.

out of curiosity, what's the source for this change? could you provide it as a reference on a new line after the Celestrak reference above?